tensorboard/webapp/widgets/data_table/column_selector_component.ng.html

```diff
@@ -25,9 +25,14 @@
 
   <div class="column-load-info">
     <label>{{ numColumnsLoaded }} columns loaded.</label>
-    <label *ngIf="hasMoreColumnsToLoad">
-      Warning: There were too many columns to load all of them.
-    </label>
+    <div *ngIf="hasMoreColumnsToLoad" class="load-more-columns">
+      <label>
+        Warning: There were too many columns to load all of them efficiently.
+      </label>
+      <button mat-stroked-button (click)="loadAllColumnsClicked()">
+        Load all anyway
+      </button>
+    </div>
   </div>
 
   <div #columnList class="column-list">
```

Comment:
We may want to wrap the list of columns in a virtual scroll viewbox prior to launching the data project
https://material.angular.io/cdk/scrolling/overview#cdk-virtual-scroll-overview